FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift

```diff
@@ -113,4 +118,57 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
         waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 15)
         XCTAssertTrue(app.staticTexts["Mozilla VPN"].exists, "The main page is not loaded correctly")
         }
-    }
+
+    func testSignInAsSubscribedUser() {
+            // The main screen is shown
```

Comment:
Nit: there is extra space for this and following lines. Look at previous test to get them equally aligned.